### PR TITLE
isDeleteable is changed to isDeletable

### PR DESCRIPTION
**Pull request by aman-tunerlabs**

### CODE REVIEW

File: module/project/templateTasks/helper.js [hunk 1]
```diff
@@ -649,7 +649,8 @@ module.exports = class ProjectTemplateTasksHelper {
                     tasks = await this.taskDocuments({
                         _id : {
                             $in : templateDocument[0].tasks
-                        }
+                        },
+                        parentId : { $exists : false }
```
Comment (aks30) [inline module/project/templateTasks/helper.js:653]: @aman-tunerlabs - Is this change required ?

Comment (aman-tunerlabs) [reply module/project/templateTasks/helper.js:653]: @aks30  Yeah required . When tested it was creating child tasks in top most tasks hierarchy but child tasks should only come inside children of a tasks.
